spec/feature_flagger/storage/redis_spec.rb

```diff
@@ -14,22 +16,34 @@
 
     describe '#has_value?' do
       context 'value is stored for given key' do
-        before { redis.sadd(key, value) }
+        before { 
+          redis.sadd(key, value)
+          redis.sadd(resource_key, key)
+        }
         it { expect(storage).to have_value(key, value) }
+        it { expect(storage).to have_value(resource_key, key) }
```

Comment:
Acho que podemos melhorar este teste aqui, pois ele diz `value is stored for given key` mas ele faz 2 expectations esperando dois valores, acho que temos 2 cenários distintos aqui;